lib/partition_setup.pm

```diff
@@ -196,14 +196,17 @@ sub unselect_xen_pv_cdrom {
 sub enable_encryption_guided_setup {
     my $self = shift;
     send_key $cmd{encryptdisk};
-    if (!get_var('ENCRYPT_ACTIVATE_EXISTING')) {
-        assert_screen 'inst-encrypt-password-prompt';
-        type_password;
-        send_key 'tab';
-        type_password;
-        send_key $cmd{next};
-        installation_user_settings::await_password_check;
+    # Bug is only in old storage stack
+    if (get_var('ENCRYPT_ACTIVATE_EXISTING') && !is_storage_ng) {
+        record_info 'bsc#993247 https://fate.suse.com/321208', 'activated encrypted partition will not be recreated as encrypted';
```

Comment:
the that you mention for soft failure has status VERIFIED / FEATURE , this mean that bug is not a bug. So you need or create another bug or replace soft-failure with normal behavior

Reply:
this is not "soft-failure" but a `record_info` so it's fine